scrapy_crawlera/middleware.py

```diff
@@ -191,6 +208,14 @@ def process_exception(self, request, exception, spider):
             self._clear_dns_cache()
             self._set_custom_delay(request, self.connection_refused_delay)
 
+    def _retry_auth(self, response, request):
+        logging.debug("Retrying crawlera request for authentication issue")
```

Comment:
@hcoura given how infrequently this occurs in production could we not change this to a **warning**? This should address your concerns around alerting users about the possibility of invalid credentials.

Reply:
I'd say to change it to, at least info, but warning looks better.

I think that, before going to production, people will run it locally at least once.
So, maybe they won't wait 10 minutes, but just check the log messages.

Reply:
warning should be fine.